sql/table.go

```diff
@@ -164,6 +164,8 @@ func encodeTableKey(b []byte, v parser.Datum) ([]byte, error) {
 		return encoding.EncodeNumericFloat(b, float64(t)), nil
 	case parser.DString:
 		return encoding.EncodeBytes(b, []byte(t)), nil
+	case parser.DNull:
+		return b, nil
```

Comment:
Postgres supports sorting NULLs first or last. I think this applies to indexes too. Why don't you think we need that property?